viadot/flows/vid_club_to_adls.py

```diff
@@ -0,0 +1,196 @@
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Literal
+import pandas as pd
+
+import pendulum
+from prefect import Flow
+from prefect.backend import set_key_value
+from prefect.utilities import logging
+
+from viadot.task_utils import (
+    add_ingestion_metadata_task,
+    df_get_data_types_task,
+    df_map_mixed_dtypes_for_parquet,
+    df_to_csv,
+    df_to_parquet,
+    dtypes_to_json_task,
+    update_dtypes_dict,
+)
+from viadot.tasks import AzureDataLakeUpload, VidClubToDF
+
+logger = logging.get_logger(__name__)
+
+
+class VidClubToADLS(Flow):
+    def __init__(
+        self,
+        name: str,
+        source: Literal["jobs", "product", "company", "survey"] = None,
+        from_date: str = "2022-03-22",
+        to_date: str = None,
```

Comment:
As I remember `to_date` was assigned in task as `today()`. Is it correct

Reply:
Actually only in task's init, it's `to_date: str = datetime.today()`, which doesn't make sense, because it's not string and in source there is already logic for `to_date == None`. 

So I can keep this as it is, and `to_date` by assigned as `datetime.today()` while executing source class or set default as `datetime.today().strftime("%Y-%m-%d")` and update defaults in task and source class.

Reply:
The flow will overwrite the task and task will overwrite the source parameter. If the flow has `None` the source will be the same.

Reply:
`to_date` parameter updated to have default value `None` and added comment in docstrings in flow, task and source classes:
"End date for the query. By default None, which will be executed as datetime.today().strftime("%Y-%m-%d") in code."